lib/linguist/heuristics.yml

```diff
@@ -476,6 +476,8 @@ disambiguations:
     pattern: '^(use |fn |mod |pub |macro_rules|impl|#!?\[)'
   - language: RenderScript
     pattern: '#include|#pragma\s+(rs|version)|__attribute__'
+  - language: XML
+    pattern: '<\?xml version="[\d.]+" encoding="[\w\d-]+"\?>'
```

Comment:
Matching `^<\?xml` should be enough. The [XML specification](https://www.w3.org/TR/REC-xml/#sec-prolog-dtd) only mandates that `<?xml` be used; accurately matching the rest of the header is more complicated than the sample might imply. We can't assume future versions of Katalon Studio will stay consistent about the order or casing of attributes (or even their inclusion, since the `encoding` attribute isn't actually mandatory).